backtrader/linebuffer.py

```diff
@@ -769,7 +769,10 @@ def _once_op(self, start, end):
         op = self.operation
 
         for i in range(start, end):
-            dst[i] = op(srca[i], srcb[i])
+            try:
+                dst[i] = op(srca[i], srcb[i])
+            except ZeroDivisionError:
```

Comment:
ZeroDivisionError indicates a bug in vast majority of cases - catching it (effectivly masking a bug) and especially setting the line data to zero is not a good fix in generic code IMHO